### ISSUE TITLE
Need streams?game to support comma delimited list

### ISSUE THREAD

**Issue by Jaxel** [justintv/Twitch-API#56]
https://github.com/justintv/Twitch-API/blob/master/resources/streams.md

The "https://api.twitch.tv/kraken/streams" supports calling of multiple channels with:

GET /streams?channel=incredibleorb,incontroltv

This works great, however the calling of games with:

GET /streams?game=Diablo+III

Does not support multiple games. It can only query for one game at a time. Would it be possible to get the game parameter to support a comma delimited list similar to how the channel parameter works?

**Comment by Jaxel** [comment]
Been over 2 years now... any updates?

**Comment by Vinlock** [comment]
Could you do something like this?

https://api.twitch.tv/kraken/streams?game=Aion:%20Steel%20Cavalry|Aion:%20Ascension|Aion:%20Dark%20Betrayal

**Comment by Jpon9** [comment]
Why not accept an array of games in the parameters?  Like this:

    https://api.twitch.tv/kraken/streams/?game[]=Counter-Strike: Global Offensive&game[]=H1Z1

That would fix any issues with games being named a certain way.  This functionality would be extremely helpful.

**Comment by TournyMasterBotCurse** [comment]
It would make more sense to be able to request the information based off the twitch game id rather than the names, that way the mapping can always be a 1:1 for outside users, and a 1:many for the various spellings of the game name.

**Comment by Jpon9** [comment]
Yeah, that would be great too, Tourny.  Just something to add this functionality back in.

**Comment by jaksah** [comment]
I'm also looking for a feature like this. Any recent updates?

**Comment by maakep** [comment]
Another year, another bump for this feature

**Comment by freaktechnik** [comment]
This belongs to https://discuss.dev.twitch.tv and not an inactive bug tracker for obsolete API documentation, if you want to really get Twitch to kniw about the need.
